sale_order_general_discount/models/sale_order_line.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    discount = fields.Float(
+        compute="_compute_general_discount",
+        store=True,
+        readonly=False,
+        string="Discount (%)",
+        digits="Discount",
+        default=0.0,
```

Comment:
Please don't redefine these already set attributes. It forces you to re-translate without need (or to be inconsistent with upstream).
```suggestion
```